src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -73,9 +73,19 @@ public void Initialize(ExtensionConfigContext context)
 
             StaticServiceHubContextStore.ServiceManagerStore = new ServiceManagerStore(options.AzureSignalRServiceTransportType, configuration, loggerFactory);
 
-            var url = context.GetWebhookHandler();
-            logger.LogInformation($"Registered SignalR trigger Endpoint = {url?.GetLeftPart(UriPartial.Path)}");
-
+            bool triggerEnabled = false;
+            try
+            {
+                var url = context.GetWebhookHandler();
+                logger.LogInformation($"Registered SignalR trigger Endpoint = {url?.GetLeftPart(UriPartial.Path)}");
+                triggerEnabled = true;
+            }
+            catch(Exception ex)
+            {
+                logger.LogWarning("SignalR trigger requires 'AzureWebJobsStorage' connection string being set. All SignalR trigger functions will be suppressed. " + 
+                    $"It's expected if you're using Azure Static Web Apps but not in other secnarios. {ex}");
```

Comment:
I don't think this should be a warning with the exception displayed. I think the majority of customers will be using only the input/output bindings and don't need to have the storage connection string set (especially when running locally). They would be alarmed if they saw this message.

Perhaps display something like:

```suggestion
                logger.LogInformation("SignalR input and output bindings are enabled. " + 
                    "To use the SignalR trigger, please set 'AzureWebJobsStorage' to a valid connection string.");
```

Is it possible to test this build of the extension in Static Web Apps?

Reply:
From my experience, if you don't set `AzureWebJobsStorage` connection string, function app will directly crash with error (in local): `Missing value for AzureWebJobsStorage in local.settings.json. This is required for all triggers other than httptrigger, kafkatrigger. You can run 'func azure functionapp fetch-app-settings <functionAppName>' or specify a connection string in local.settings.json.` 
So, I think a warning makes sense because customer usually set `AzureWebJobsStorage` by default.

For the testing, does Static Web App support registering customer dependencies other than `bundles`. If it does, I can do a test

Reply:
Checked Kusto and 40% of deployed function apps that use SignalR bindings have only http triggers (and therefore would get this warning if running without storage). That's why I'm concerned about how this message appears. Static web apps allow only HTTP triggers so their function apps fall into this category as well.

I'm actually not sure how to test this in Static Web Apps. Perhaps it's enough to run it in a regular function app with no storage account configured.

Reply:
Then I think log as information is enough. I've already tested it at where has no `AzureWebJobsStorage`